src/main/target/FLIGHTONE/target.h

```diff
@@ -0,0 +1,144 @@
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define TARGET_BOARD_IDENTIFIER "RVLT"
+#define USBD_PRODUCT_STRING     "RevoltOSD"
+#define TARGET_DEFAULT_MIXER    MIXER_QUADX_1234
+
+
+
+#define LED0_PIN                PB5  
+
+#define USE_BEEPER
+#define BEEPER_PIN              PB4
+
+#define ENABLE_DSHOT_DMAR       true
+
+#define USE_TRANSPONDER
+
+#define DEFAULT_RX_FEATURE      FEATURE_RX_SERIAL
+
+#define USE_TARGET_CONFIG
+
+#define USE_SERIAL_4WAY_BLHELI_INTERFACE
+
+
+#define INVERTER_PIN_UART1      PC0
+
+/*----------Spi Config--------*/
+#define USE_SPI
+
+#define USE_SPI_DEVICE_1
+#define SPI1_NSS_PIN            PA4
+#define SPI1_SCK_PIN            PA5
+#define SPI1_MISO_PIN           PA6
+#define SPI1_MOSI_PIN           PA7
+
+#define USE_SPI_DEVICE_2
+#define SPI2_NSS_PIN            PB12
+#define SPI2_SCK_PIN            PB13
+#define SPI2_MISO_PIN           PB14
+#define SPI2_MOSI_PIN           PB15
+
+/*----------I2C Config--------*/
+#define USE_I2C
+#define USE_I2C_DEVICE_1
+#define I2C_DEVICE              (I2CDEV_1)
+#define I2C1_SCL                PB8
+#define I2C1_SDA                PB9
+
+/*----------Gyro Config--------*/
+#define MPU6000_CS_PIN          PA4
+#define MPU6000_SPI_INSTANCE    SPI1
+
+#define MPU6500_CS_PIN          PA4
+#define MPU6500_SPI_INSTANCE    SPI1
+
+#define USE_GYRO
+#define USE_ACC
+
+#define USE_GYRO_MPU6500
+#define USE_GYRO_SPI_MPU6500
+#define GYRO_MPU6500_ALIGN      CW0_DEG
+
+#define USE_ACC_MPU6500
+#define USE_ACC_SPI_MPU6500
+#define ACC_MPU6500_ALIGN       CW0_DEG
+
+
+#define USE_EXTI
+#define MPU_INT_EXTI            PC4
+#define USE_MPU_DATA_READY_SIGNAL
+
+/*----------Flash Config--------*/
+#define M25P16_CS_PIN           PB3
+#define M25P16_SPI_INSTANCE     SPI2
+#define USE_FLASHFS
+#define USE_FLASH_M25P16
+
+/*----------OSD Config--------*/
+#define USE_MAX7456
+#define MAX7456_SPI_INSTANCE    SPI2
+#define MAX7456_SPI_CS_PIN      PD2
+#define MAX7456_SPI_CLK         (SPI_CLOCK_STANDARD) // 10MHz
+#define MAX7456_RESTORE_CLK     (SPI_CLOCK_FAST)
+
+/*----------Uart Config--------*/
+#define USE_UART1
+#define UART1_RX_PIN            PA10
+#define UART1_TX_PIN            PA9
+#define UART1_AHB1_PERIPHERALS  RCC_AHB1Periph_DMA2
+
+#define USE_UART3
+#define UART3_RX_PIN            PB11
+#define UART3_TX_PIN            PB10
+
+#define USE_UART4
+#define UART4_RX_PIN            PA1
+#define UART4_TX_PIN            PA0
+
+#define USE_UART6
+#define UART6_RX_PIN            PC7
+#define UART6_TX_PIN            PC6
+
+#define USE_SOFTSERIAL1
```

Comment:
You could set default pins for `SOFTSERIAL` 1 and 2 here if your wiring diagram includes them:
```
#define SOFTSERIAL1_RX_PIN
#define SOFTSERIAL1_TX_PIN
```